src/filemenu/filemenu_info.c

```diff
@@ -2,6 +2,14 @@
 #include "filemenu.h"
 #include "hud_element.h"
 
+#if VERSION_CN
+#define CREATE_SUCCESS_NUMBER_X 49
+#define NUMBER_OFFSET_Y 1
+#else
+#define NUMBER_OFFSET_Y 0
```

Comment:
Better to define the position as 6 here and dispense with the offset. I think eventually well want to parameterize these UI positions with defines anyway.

Reply:
I don't think that makes sense, combined with the other 'widespread' uses of 'NUMBER_OFFSET_Y' being used across filemenu